internal/tools/tools.go

```diff
@@ -37,6 +37,8 @@ import (
 	_ "golang.org/x/tools/cmd/goimports"
 	// mockgen for generating mocks
 	_ "github.com/golang/mock/mockgen"
+	// gowrap for generating decorators for interface
+	_ "github.com/hexdigest/gowrap"
 	// mockery for generating mocks
```

Comment:
oh, actually, I didn't mean to include this one.  it was just for testing.
lemme undo that.